RInAppMessaging/Classes/Protocols/HttpRequestable.swift

```diff
@@ -123,12 +125,13 @@ extension HttpRequestable {
             let statusCode = (response as? HTTPURLResponse)?.statusCode ?? 0
 
             guard 100..<400 ~= statusCode,
-                let dataToReturn = data,
-                let serverResponse = response as? HTTPURLResponse else {
+                  let dataToReturn = data,
+                  let serverResponse = response as? HTTPURLResponse else {
 
-                    completion(.failure(.httpError(statusCode, response, data)))
-                    print("InAppMessaging: HTTP call failed (\(statusCode))")
-                    return
+                completion(.failure(.httpError(statusCode, response, data)))
+                let errorMessage = data != nil ? String(data: data!, encoding: .utf8) : ""
+                print("InAppMessaging: HTTP call failed (\(statusCode))\n\(errorMessage ?? "")")
```

Comment:
Should this be a `print`?

Reply:
oh. good question 👍

Reply:
Ok I remember now. I wanted this to be always printed in the logs and `reportError` is used in services that extend this protocol